packages/query/src/bitcoin/ordinals/inscriptions.query.ts

```diff
@@ -193,12 +171,19 @@ export function useGetInscriptionsInfiniteQuery<T extends unknown = GetInscripti
         addressesMap,
       };
     },
-    getNextPageParam(prevInscriptionsQuery) {
+    initialPageParam: {
+      addressesMap: {},
+      addressesWithoutOrdinalsNum: 0,
+      fromIndex: 0,
+      offset: 0,
+      total: 0,
+    },
+    getNextPageParam(lastPage) {
       const { offset, total, stopNextFetch, fromIndex, addressesWithoutOrdinalsNum, addressesMap } =
-        prevInscriptionsQuery;
+        lastPage;
       if (stopNextFetch) return undefined;
 
-      // calculate offset for next fetch
+      // Calculate offset for next fetch
```

Comment:
For this, it could be nice to extract into a function - `calculateOffset`. 

Not something necessary to bundle in right now but calling it as I see it and adding as a suggestion for how trying to avoid using `let` can force a more understandable style

```
function calculatedOffset(offset, inscriptionsLazyLoadLimit, total) {
  const initialOffset = offset + inscriptionsLazyLoadLimit; 
 const offsetAboveLimit = offset + inscriptionsLazyLoadLimit > total;

   if (offsetAboveLimit) {
       return offset + (total - offset);
      }
 return initialOffset;

}

```